packages/pasteup/palette.ts

```diff
@@ -115,19 +117,23 @@ const neutral = {
     20: '#333333',
     46: '#767676',
     60: '#999999',
+    85: '#d9d9d9',
     86: '#dcdcdc',
     93: '#ededed',
     97: '#f6f6f6',
     100: '#ffffff',
 };
 
-const specialReport = { dark: '#3f464a' };
-
-const labs = {
-    dark: '#65a897',
-    main: '#69d1ca',
+const labs: PillarColours & {} = {
```

Comment:
what does the `& {}` mean? Haven't seen this before and my typescript foo is weak!

Reply:
Oops! I wasn't sure if PillarColours type would match to what was needed for Labs, so was going to add some extra info to the interface and forgot to remove. Thanks for catching!